### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -27,14 +27,11 @@ ynh_script_progression --message="Restoring the data directory..." --weight=1
...
+# INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing Peertube remote runner npm module..." --weight=1
+ynh_script_progression --message="Installing dependencies..."
 
-pushd "$install_dir"
-  ynh_use_nodejs
-  ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_npm install @peertube/peertube-runner
-popd
+ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
```
Comment (lapineige) [inline scripts/restore:34]: Why ?
Does that change the functionality ?

File: scripts/restore [hunk 2]
```diff
@@ -53,20 +50,10 @@ yunohost service add $app --description="Peertube remote runner" --log="/var/log
...
 
 ynh_systemd_action --service_name=nginx --action=reload
 
-#=================================================
-# RUNNER REGISTRATION
-#=================================================
-
-pushd "$install_dir"
-  ynh_use_nodejs
-  ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_node "$install_dir/node_modules/@peertube/peertube-runner" register --url $peertube_url --registration-token $runner_token --runner-name $app\_$runner_id
```
Comment (lapineige) [inline scripts/restore]: Why remove that ?

File: manifest.toml [hunk 3]
```diff
@@ -2,8 +2,8 @@ packaging_format = 2
 
 id = "peertube_remote_runner"
 name = "PeerTube Remote Runner"
-description.en = "Remote runner to offload high CPU/long job (such as transcoding) from Peertube main server"
```
Comment (lapineige) [inline manifest.toml:5]: I'm against this change, the role of that app is already complicated enough, more explicit explanations are needed.

File: scripts/restore (hunk 2 above)
Comment (ericgaspar) [reply scripts/restore]: Restore script is not meant to be install script

File: scripts/restore (hunk 1 above)
Comment (ericgaspar) [reply scripts/restore:34]: Restore script is not meant to be install script. we are restoring `install_dir` path with peertube-runner already installed in the install script

File: manifest.toml (hunk 3 above)
Comment (ericgaspar) [reply manifest.toml:5]: The `DESCRIPTION.md` file is the place you are looking for to provide proper explanation.

File: scripts/restore (hunk 1 above)
Comment (lapineige) [reply scripts/restore:34]: I saw the rest of you changes afterwards, now I understand them :)

However in that case restoring or install is basically the same (except install might have a never version), as there is no particular thing to backup, so I wonder if the restore script is even necessary :thinking:

File: manifest.toml (hunk 3 above)
Comment (lapineige) [reply manifest.toml:5]: But it's not that long, people won't look at the description that easily, and this one doesn't convey a lot of meaning if you don't already know the purpose if this app :(
I feel like it's a step backward :/

Comment (lapineige) [reply manifest.toml:5]: So I tried to find a middle ground by extending it a bit, but not much, and convey more meaning in the French version (which was clearly worse than the English one)

**Pull request merged**